pkg/controller/status/status.go

```diff
@@ -163,6 +163,9 @@ func (c *Controller) merge(existing *configv1.ClusterOperator) *configv1.Cluster
 				degradingFailure = false
 			} else {
 				klog.V(3).Infof("Number of last gather failures %d exceeded the threshold %d. Marking as degraded.", summary.Count, GatherFailuresCountThreshold)
+				degradingFailure = false
+				disabledReason = summary.Reason
+				disabledMessage = summary.Message
```

Comment:
Well we need to update the log messages as well right? The condition can be simplified IMO. We always can set `degradingFailure = false`.

Reply:
it was so simple that I overlooked these, thx :D